src/views/virtualmachines/tree/VirtualMachineTreeView.tsx

```diff
@@ -44,14 +45,16 @@ const VirtualMachineTreeView: FC<VirtualMachineTreeViewProps> = ({
   const [drawerWidth, setDrawerWidth] = useLocalStorage(TREE_VIEW_LAST_WIDTH, OPEN_DRAWER_SIZE);
   const [drawerOpen, setDrawerOpen] = useLocalStorage(SHOW_TREE_VIEW, SHOW);
 
-  const { featureEnabled: treeViewEnabled, loading } = useFeatures(TREE_VIEW);
+  const { featureEnabled: treeViewFeatureEnabled, loading } = useFeatures(TREE_VIEW);
+
+  const hideTreeview = !treeViewFeatureEnabled && !runningTourSignal.value;
```

Comment:
if the treeview feature is enabled but the tour is not running, won't this mean the tree view won't be shown? why we need this condition?

Reply:
Maybe the condition is a little bit confusing but its correct.
THe treeview will be hided when the feature is disabled and its not running a guided tour. 
The inverse would be to show the treeview if the feature is enabled and is running a guided tour. In all those two cases